### PR TITLE
Drop deprecated 'p-*' selectors

### PR DESCRIPTION
**Pull request by nmichaud**
Finishes #9. 

This will require careful review.

### CODE REVIEW

**Comment by jasongrout** [comment]
Thanks.

I think the big decision that wasn't clear from #9 is: is this a change requiring a major version Lumino? If so, I think this shouldn't be the *only* change in Lumino 2, so what is the larger story about what Lumino 2 is going to be?

**Comment by afshin** [comment]
This requires a major version bump, I think.

I don't think we *must* have other API breakages here. This could simply be a transitional version. I think releasing a version of Lumino that has no deprecated APIs is enough of a win, personally.

**Comment by blink1073** [comment]
How would this affect ipywidgets 7?

File: packages/widgets/tests/src/commandpalette.spec.ts [hunk 1]
```diff
@@ -686,9 +686,6 @@ describe('@lumino/widgets', () => {
         it('should create the full class name for the item node', () => {
           let name = renderer.createItemClass({ item, indices: null, active: false });
           let expected = 'lm-CommandPalette-item testClass';
-          /* <DEPRECATED> */
-          expected = 'lm-CommandPalette-item testClass';
```
Comment (afshin) [inline packages/widgets/tests/src/commandpalette.spec.ts]: Looks like this was a typo to begin with?

File: packages/widgets/tests/src/commandpalette.spec.ts [hunk 2]
```diff
@@ -697,9 +694,6 @@ describe('@lumino/widgets', () => {
           toggledFlag = true;
           let name = renderer.createItemClass({ item, indices: null, active: true });
           let expected = 'lm-CommandPalette-item lm-mod-disabled lm-mod-toggled lm-mod-active testClass';
-          /* <DEPRECATED> */
-          expected = 'lm-CommandPalette-item lm-mod-disabled lm-mod-toggled lm-mod-active testClass';
```
Comment (afshin) [inline packages/widgets/tests/src/commandpalette.spec.ts]: Same here?

File: packages/widgets/tests/src/commandpalette.spec.ts (hunk 1 above)
Comment (nmichaud) [reply packages/widgets/tests/src/commandpalette.spec.ts]: Yeah both of these look like typos in the initial PR.

**Comment by afshin** [comment]
Closing this PR because it is implemented in [this commit](https://github.com/jupyterlab/lumino/pull/319/commits/0a8b1f61534b403eec2832ad44ccf93237f65e16) (authored by @afshin and @nmichaud) in #319

**Pull request closed without merging**